.pipelines/e2e.yml

```diff
@@ -1,3 +1,19 @@
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    exclude:
+    - docs/*
```

Comment:
Explicit run on push on `master` with omit of `docs/*` folder. Implicit is `master` all.